### PR TITLE
feat(dropdowns): use svg icon

### PR DESCRIPTION
**Pull request by jefflembeck**
Update that includes using svg icon for dropdown caret +
make sure that the documentation is updated and better looking

screen shot 2016-06-01 at 12 03 54 pm

### CODE REVIEW

**Comment by jefflembeck** [comment]
cc @stubbornella @jerrygabra how do we feel about this spacing?

File: src/pivotal-ui/components/dropdowns/dropdowns.scss [hunk 1]
```diff
@@ -381,6 +397,18 @@ This is a notification dropdown with alerts.
   }
 }
 
+.dropdown-toggle {
+  .caret {
+    display: inline-block;
+    border: 0;
+    margin-right: -0.6em;
```
Comment (stubbornella) [inline src/pivotal-ui/components/dropdowns/dropdowns.scss]: is this always going to match the font correctly? Did it look right with a small button (that has smaller font)?

File: src/pivotal-ui/components/dropdowns/dropdowns.scss [hunk 2]
```diff
@@ -381,6 +397,18 @@ This is a notification dropdown with alerts.
...
+    padding-top: 7px;
+    margin-left: 5px;
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 20px;
```
Comment (stubbornella) [inline src/pivotal-ui/components/dropdowns/dropdowns.scss]: is it necessary to set the font-size now that we are using svg?

**Comment by stubbornella** [comment]
looks good, I left a couple small comments

**Comment by jefflembeck** [comment]
pivotal ui css dropdowns 2016-06-01 12-27-50
pivotal ui css dropdowns 2016-06-01 12-27-59

File: src/pivotal-ui/components/dropdowns/dropdowns.scss (hunk 2 above)
Comment (jefflembeck) [inline src/pivotal-ui/components/dropdowns/dropdowns.scss]: `width` and `height` are proportionate based upon the `icon` class. In the case of the `caret`, here. I'm inflating a bit. I've changed to `1.25em` in the most recent push. Makes it way better for proportional changes.

File: src/pivotal-ui/components/dropdowns/dropdowns.scss (hunk 1 above)
Comment (jefflembeck) [inline src/pivotal-ui/components/dropdowns/dropdowns.scss]: Small change on it, but yes!

**Comment by jefflembeck** [comment]
added more docs (for small button) so we can catch regressions in that in the future

**Comment by jefflembeck** [comment]
got the 👍 from @jerrygabra

**Pull request merged**